config/containers/resource_constraints.md

```diff
@@ -111,9 +111,9 @@ Its setting can have complicated effects:
   [Prevent a container from using swap](#prevent-a-container-from-using-swap).
 
 - If `--memory-swap` is unset, and `--memory` is set, the container can use
-  twice as much swap as the `--memory` setting, if the host container has swap
+  as much swap as the `--memory` setting, if the host container has swap
   memory configured. For instance, if `--memory="300m"` and `--memory-swap` is
-  not set, the container can use 300m of memory and 600m of swap.
+  not set, the container can use 300m of memory and 300m of swap.
```

Comment:
"container can use 600m in total of memory and swap" ?

https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt

Reply:
yes.  memory + swap = 600m

Reply:
so swap can be more than 300m, right? The current text seems confusing

Reply:
Usually we don't directly say how many swaps are available; and the limitations of cgroups also limit memory usage and memory + swap usage.

Reply:
>so swap can be more than 300m,  right?

In my understanding, it should be a maximum of 300m.

Reply:
My initial change here was to correct it;
"container can use 600m in total of memory and swap" is more suitable, but compared to other examples, it may not be straightforward.

Reply:
Not straightforward, but it's kernel's limitation

https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt
```
* why 'memory+swap' rather than swap.
The global LRU(kswapd) can swap out arbitrary pages. Swap-out means
to move account from memory to swap...there is no change in usage of
memory+swap. In other words, when we want to limit the usage of swap without
affecting global LRU, memory+swap limit is better than just limiting swap from
an OS point of view.
```

Reply:
Although I think this is explained from a **design perspective**. 

I think your suggestion is reasonable :+1: and this can avoid some confusion.
Let me update this text.